Extending/Packages/Installing-And-Uninstalling-Packages/index.md

```diff
@@ -0,0 +1,144 @@
+
+![Backoffice - removing content](images/removing-content.png)
+
+#### Remove Media provided by the package
+Similar to Content, Media also might have to be removed.
+
+![Backoffice - removing media](images/removing-media.png)
+
+#### Remove Document Types
+Document Types can be removed from the **Settings** section. If fully removing the package, all Document Types can be deleted, as there are no default Document Types in a clean-slate Umbraco installation.
+
+![Backoffice - removing document types](images/removing-document-types.png)
+
+#### Removing Data Types
+As opposed to Document Types, there are some Data Types that are available out of the box when Umbraco is installed. It is not recommended to remove them. The safe approach is to delete any item that starts with a Document Type prefix and includes multiple dashes. That is the default naming convention for new configurations of Data Types (e.g. Blog - How many posts should be shown - Slider)
+
+![Backoffice - removing data types](images/removing-datatypes.png)
+
+#### Removing Templates
+No templates are available out of the box in a new installation. If cleaning up after a package, it would be okay to delete all that are present
+
+![Backoffice - removing templates](images/removing-templates.png)
+
+#### Removing Partial Views
+Out of the box, there are a few views available in the *blocklist* and *grid* folders. Everything else can theoretically be removed.
+
+![Backoffice - removing partial views](images/removing-partials.png)
+
+### Cleaning leftover files on disk
+
+Some packages might reference other items. For example, installing The Starter Kit also adds `Bergmania.OpenStreetMap` to your project. That component will show up as installed in the backoffice even after uninstalling the NuGet package.
+
+![Backoffice - Packages section - leftover dependency](images/installed-package-leftovers-backoffice.png)
+
+In many cases, custom dashboards, editors, and scripts are left in the `App_Plugins` folder after a package has been uninstalled via NuGet. These files also have to be deleted manually.
+
+![Visual Studio - App Plugins leftover files](images/app-plugins-starterkit.png)
+
+:::note
+Please note this particular guide targets a specific package. There are many packages out there, and each one is different. The exact steps presented here might not work the exact same way for all the packages, though the general approach should still apply.
+:::
+
+## Uninstalling packages like SEO Checker
+
+More advanced packages that add functionality on top of Umbraco usually rely on providing custom, compiled code. That being said, many of such packages also implement custom sections, dashboards, editors, and views.
+
+In this example, we are using SEO Checker which is not a very invasive package out of the box. It allows the developers of the site to add custom properties to Document Types used to track good search engine optimization practices.
```

Comment:
I think this sentence should be rephrased.
"Is not very invasive" - what does that mean?

Perhaps just something like:
"In this example, we will be using SEO Checker which allows developers of the site to add custom properties to Document Types used to track search engine optimization practices.

Reply:
it means it doesn't modify any underlying functionality and instead adds some new functionality.
Any idea how to phrase it better? 🤔

Reply:
I did give you an example, yes ;) 

> Perhaps just something like:
"In this example, we will be using SEO Checker which allows developers of the site to add custom properties to Document Types used to track search engine optimization practices.